### ISSUE TITLE
Dask Demo Day - Sign up issue

### ISSUE THREAD

**Issue by ncclementi** [dask/community#307]
Next Demo Day: December 5th
---

See what the Dask community has been up to, or share some Dask work of your own. Demos are short and informal (~5-10 minutes). Have something you'd like to share? Leave a comment and let us know.

Meetings are held on the first Thursday of each month at 11am US Eastern on Zoom: https://dask.org/meeting-room

Subscribe to the Dask event calendar to be notified of changes:
- [Google Calendar](https://calendar.google.com/calendar/u/0?cid=NGwwdnRzMGMxY2dkYnE1amhjb2dqNTVzZnNAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
- [iCal](https://calendar.google.com/calendar/ical/4l0vts0c1cgdbq5jhcogj55sfs%40group.calendar.google.com/public/basic.ics)

Past Demo Days
---

Meetings are recorded and shared on the [Dask YouTube channel](https://youtube.com/playlist?list=PLJ0vO2F_f6OD0cwscL_yeG1aC3ESBquAT&si=XudNPWybgR21zIm8).

### Oct 3rd, 2024
- @rjzamora "pin and persist" 

- ### Sept 5th, 2024 [youtube recording](https://youtu.be/afmus2EvccU)
  - @dougbrn [Dask Applications in Astronomy with LINCC Frameworks](https://github.com/lincc-frameworks/notebooks_lf/blob/main/dask_demo_day/dask_demo_day_lincc.ipynb)
  - @jacobtomlinson [dask-jobqueue](https://jobqueue.dask.org/en/latest/) demo

- ### August 1st, 2024
  - @raybellwaves [github-repo-explorer](https://github.com/raybellwaves/github-repo-explorer)

- ### June 6th, 2024
  - @ahuang11 [streamjoy](https://ahuang11.github.io/streamjoy/)
  - @v2thegreat hvPlot
  
- ### March 21st, 2024 [youtube recording](https://youtu.be/HTKzEDa2GA8)
  - @fjetter Dask DataFrame improvements
  - @mrocklin Large scale population of vector databases for RAG 
  - @jrbourbeau Easy GPU access with Coiled
 
- ### February 15th, 2024 [youtube recording](https://www.youtube.com/watch?v=07e1JL83ur8)
  - @mrocklin One trillion row challenge
  - @jacobtomlinson deploying Dask on Databricks with [dask-databricks](https://github.com/dask-contrib/dask-databricks)
  - @jrbourbeau deploying Prefect workflows on the cloud with Coiled
  - @quasiben scaling embedding pipelines (LlamaIndex + Dask)
  - @ntabris using AWS Cost Explorer to see the cost of public IPv4 addresses

- ### January 18th, 2024 [youtube recording](https://www.youtube.com/watch?v=wkQzVNQdgW0)
  - @cisaacstern (and maybe @jacobtomlinson) Apache Beam DaskRunner
  - @mrocklin Array expressions
  - @scharlottej13 one billion row challenge

- ### October 19th, 2023 [youtube recording](https://www.youtube.com/watch?v=5YXiRl41DKA)
  - @mrocklin TPC-H benchmarks for Spark, Dask, Polars, DuckDB
  - @mrchtr [Fondant](https://github.com/ml6team/fondant)
  - @jhamman Dask <> [Arraylake](https://docs.earthmover.io/) integration
  - @jacobtomlinson "Who uses RAPIDS?"

- ### September 21st, 2023 [youtube recording](https://www.youtube.com/watch?v=-3pk5B-9QDQ)
  - @fjetter Performance with P2P array rechunking
  - @phofl Dask expressions
  - @scharlottej13 + @dcherian Processing a quarter petabyte geospatial dataset in the cloud

- ### August 17th, 2023 [youtube recording](https://www.youtube.com/watch?v=MepdrbDjYPc)
  - @fjetter -- distributed + memray integration for memory profiling
  - @mrocklin -- coiled setup CLI
  - @jrbourbeau -- Make Dask and Earthaccess work well together

- ### July 20th, 2023 [youtube recording](https://www.youtube.com/watch?v=dTsTkRD6rbY)
    - @hendrikmakait  Shuffle resilience  
    - @Matt711 : Dask-Kubernetes update 
    - @GueroudjiAmal  external tasks in Dask distributed (https://github.com/GueroudjiAmal/distributed)
    - @skrawcz Dask <> [Hamilton](https://github.com/dagWorks-Inc/hamilton) integration

- ### June 15th, 2023 [youtube recording](https://www.youtube.com/watch?v=eUnvdAoMibE)
    - dask-geopandas demo by @martinfleis 
    - Fine performance dask metrics @crusaderky 
    - Gil monitoring on dask @milesgranger  
    
- ###  May 18th, 2023 [youtube recording](https://www.youtube.com/watch?v=tyk7rwR8ifE)
    - Dask Expressions - @rjzamora 
    - deltadask - @MrPowers  
    - ydata-profiling -  @miriamspsantos / @fabclmnt 
    - Dask-bigquery - @ncclementi /@j-bennet

- ### April 20th, 2023 -[youtube recording](https://www.youtube.com/watch?v=7ja_YHYg23A)
    - dask-awkward and dask-histogram for high energy physics analysis @lgray (10 min)
    - daskqueue : a dask-based distributed task queue. @AmineDiro (5-7min)
    - Pyarrow strings in Dask DataFrames - @jrbourbeau (5-7 min)
    -  Launching a Jupyter/Dask cluster on NVIDIA Base Command Platform - @jacobtomlinson (5-7min)

- ### March 16th, 2023 - [youtube recording](https://www.youtube.com/watch?v=-clDr7dqONE)
    - Analysing Terabytes of Ocean Simulation model output with Xarray, xgcm and xhistogram @TomNicholas 
    - P2P shuffling @hendrikmakait 
    - Scaling weather radar data analysis with Dask @mgrover1 
    - Automatic package synchronization in Coiled Dask Clusters @dchudz 
    - Graph Neural Networks training with Dask @VibhuJawa

**Comment by ncclementi** [comment]
- @brian-methodical Can we get some folks from your team to sign up for one of the upcoming dates? 

- @mgrover1 Do you think the 03/16 date would work for you? Do you want to drop a comment with a proposed date and title?

- @TomNicholas Whenever you can drop a comment on what date would be more convenient for you, that would be great. 

- @raybellwaves @jorisvandenbossche @tastatham @martinfleis Can we get a demo (or two) that involves dask-geopandas? Or can you volunteer someone :) ?

**Comment by martinfleis** [comment]
@ncclementi I could do a short demo of dask-geopandas in May but happy to leave the place to others if they're interested.

**Comment by ncclementi** [comment]
@martinfleis I'll take you up for May, if you know of folks that would like to show their applications on any of the dates, please send them this issue. It'll be great to hear more about how dask-geopandas is being used.

**Comment by TomNicholas** [comment]
I think 16th March is good for me!

**Comment by ncclementi** [comment]
@TomNicholas Thanks Tom 🙌 , do you have a title that would summarize your demo? If not yet, we can set it later

**Comment by TomNicholas** [comment]
Something like "Analysing Terabytes of Ocean Simulation model output with Xarray, xgcm and xhistogram"

**Comment by dchudz** [comment]
@ncclementi I'd like to demo Coiled's "package sync" feature (should be interesting even for folks not planning to use Coiled).

**Comment by mgrover1** [comment]
Yes to 3/16! Still trying to come up with a title, but it will be related to scaling weather radar data analysis with dask!

**Comment by ncclementi** [comment]
Thanks @mgrover1 and @dchudz We'll have you on the list! 🎉

**Comment by ncclementi** [comment]
@martindurant and @douglasdavis would you be interested in presenting at the April upcoming session?

**Comment by lgray** [comment]
Hi @ncclementi - I'm using dask-awkward and dask-histogram for high energy physics analysis and can give some real-world examples of how we'd use @martindurant and @douglasdavis's work. I think I can fit that all into 10 minutes. I'm available for April 20th.

**Comment by ncclementi** [comment]
@lgray That would be awesome! We usually do 5-7 min but since we do not have folks for April yet, I think 10 min will work, plus you will be showcasing two dask projects! 
I'll write you down for April 20th then!!

**Comment by AmineDiro** [comment]
Hello @ncclementi , I built [daskqueue](https://github.com/AmineDiro/daskqueue), a lightweight Distributed Task Queue library built on top of Dask. Daskqueue also implements persistent queues for holding tasks on disk and surviving Dask cluster restart. I would love to present this library according to the format of your choosing and April 20th sounds excellent for me. 

The title of the talk would be **daskqueue : a dask-based distributed task queue.**

**Comment by brian-methodical** [comment]
We can take May Demo, I’ll bring Dan from my MLOps team to this weeks
meeting to intro and get involved

**Comment by ncclementi** [comment]
@AmineDiro Thank you, that sounds awesome. I'll set you on the list for April 20th, regarding the format. 

> These are 5-7 minute demos that show off ongoing or lesser-known work. We hope to have 3-5 of these during the meeting. Meetings will be recorded and advertised on social. Hopefully, this helps to educate folks on some of the great work people are up to.

I just updated this issue with a link to the meeting for clarity. I will eventually create an April issue specific.

**Comment by brian-methodical** [comment]
> We can take May Demo, I’ll bring Dan from my MLOps team to this weeks meeting to intro and get involved


RE https://github.com/dandawg

**Comment by ncclementi** [comment]
@danwang if you can drop a title for your may demo that would be great.

**Comment by dandawg** [comment]
> @danwang if you can drop a title for your may demo that would be great.

Thanks @ncclementi . I don't have a title yet--I need to narrow my topic area a little more. I'll commit to having one though by next week.

**Comment by jacobtomlinson** [comment]
@ncclementi if you're still looking for another demo for the April Demo Day I'd love to do a quick demo of launching a Jupyter/Dask cluster on NVIDIA Base Command Platform which comes with all the GPU stuff configured.

**Comment by ncclementi** [comment]
@jacobtomlinson That would be great! I'll add you to the list.

**Comment by mrocklin** [comment]
I'm talking to @fab_clemente here from https://github.com/ydataai/ydata-profiling , a dataframe profiling library (gives summary statistics and more like https://ydata-profiling.ydata.ai/examples/master/census/census_report.html) that works well with Pandas and Spark.  It might be an interesting demo to try to see if there is interest in a Dask integration.  

@ncclementi what do you think?

**Comment by ncclementi** [comment]
@fabclmnt would you be up for demoing according to the format described above on May 18th 11am EDT?

**Comment by mrocklin** [comment]
@rjzamora want to give a demo on dask-expr?

**Comment by dandawg** [comment]
I owe an update here. My topic is still in dev, and needs some more exploring. Client work has gotten in the way a bit. I'll attend the community meeting tomorrow, and I can give an update on what I'm thinking about, and maybe get some feedback.

**Comment by ncclementi** [comment]
@martinfleis Are you still on for the May Dask Demo Day? Do you have a title/topic?

**Comment by martinfleis** [comment]
@ncclementi I apparently messed up time zones (again!) and am supposed to speak at another event at the same time. @jorisvandenbossche @jsignell any chance you'd be willing to take over? If not we may need to reschedule for another month. Sorry for complications!

**Comment by jsignell** [comment]
I can't make that Demo Day unfortunately.

**Comment by rjzamora** [comment]
>@rjzamora want to give a demo on dask-expr?

Yes - I can walk through a high-level dask-expr demo :)

**Comment by ncclementi** [comment]
@martinfleis Can I move you to the June Day?

**Comment by martinfleis** [comment]
@ncclementi yes. And sorry!

**Comment by ncclementi** [comment]
@dandawg If it's still possible for you to make the May 18th demo, that would be great. So far haven't been able to find replacements.

**Comment by davemlz** [comment]
Hi @ncclementi! A date on July would work perfect for me, let me know if you have any available dates on that month! :)

**Comment by ncclementi** [comment]
@davemlz Absolutely! In July, it would be the 20th, let me know when you have a title/topic, and I'll edit the main post.

**Comment by ncclementi** [comment]
@dandawg can you give us an update on your demo for May 18th, it'll be great if you can make it. Since we do not have many folks registered.

**Comment by ncclementi** [comment]
@Lkruitwagen Would you be up for demoing in 5-7 min (see format in top comment) the work you presented here https://twitter.com/LucasKruitwagen/status/1648881135974514688

**Comment by MrPowers** [comment]
@ncclementi - I'd be happy to demo [deltadask](https://github.com/MrPowers/deltadask) on May 18th if there is still space for another speaker!  Let me know!

**Comment by ncclementi** [comment]
@MrPowers absolutely!, I'll update the comments to get you in

**Comment by mrocklin** [comment]
Excellent idea.  Thanks Powers!

**Comment by fabclmnt** [comment]
> @fabclmnt would you be up for demoing according to the format described above on May 18th 11am EDT?

Hi @ncclementi yes I would be for sure.

**Comment by ncclementi** [comment]
@fabclmnt Awesome, I'll add you to the schedule.

**Comment by crusaderky** [comment]
Adding fine performance metrics to June 15th

**Comment by milesgranger** [comment]
GIL monitoring for June 15th. :)

**Comment by Matt711** [comment]
I'd like to do a demo of the Dask Operator. Maybe talk about what's new since @jacobtomlinson 's demo

**Comment by GueroudjiAmal** [comment]
I would like to do a demo of introducing external tasks in Dask distributed on July 20th or the next date.

**Comment by ncclementi** [comment]
@davemlz I have you for a demo on July 20th. Do you have a title or topic?

**Comment by davemlz** [comment]
@ncclementi, I'll have to cancel my talk. I apologise for any inconvenience.

**Comment by skrawcz** [comment]
Thanks for chatting with me @ncclementi, I'd like to confirm that I should be able to give a quick demo of the (naive) integration [Hamilton](https://github.com/dagWorks-Inc/hamilton) has with Dask to get thoughts and reactions.

**Comment by ncclementi** [comment]
@skrawcz Awesome, I'll add you to the list!

**Comment by fjetter** [comment]
Next session I can talk about

### Performance with P2P array rechunking

- Status quo of the two available methods tasks vs p2p
- Discuss how we look at performance, what kind of metrics we are interested in and what kind of performance we expect _theoretically_
- Depending on time, shed light on some already known problems

**Comment by scharlottej13** [comment]
@mrchtr I have you down for the Oct. 19th demo day, thanks! Excited to learn more about [fondant](https://github.com/ml6team/fondant)

**Comment by jhamman** [comment]
If there's room, I'd be up for sharing a Dask <> [Arraylake](https://docs.earthmover.io/) integration demo on Oct. 19th.

**Comment by mrocklin** [comment]
Woo

**Comment by jrbourbeau** [comment]
Sounds great, thanks @jhamman

**Comment by cisaacstern** [comment]
Is this still the place to sign up for demo day?

If so, and assuming there's space in the schedule for 12/21, I'd love to present a status update on the Apache Beam `DaskRunner` (possibly with assistance of @jacobtomlinson, assuming he'll be in attendance).

**Comment by mrocklin** [comment]
It is still the place!  However, I think we're probably going to pass on
12/21 due to the proximity to the holidays.  Are you likely to still be
interested in January?

**Comment by cisaacstern** [comment]
Absolutely. Based on https://docs.dask.org/en/stable/support.html#discussion, looks like the next one will be Jan 18?

**Comment by scharlottej13** [comment]
@ahuang11 is up for presenting on [streamjoy](https://ahuang11.github.io/streamjoy/) for the next demo day on April 18th 🎉. Streamjoy turns images into animations and uses Dask for parallel image processing.

**Comment by v2thegreat** [comment]
Hi!

If spots are still available, I'd like to demo Hvplot on May 16th. I'll be showing some examples of its use with Xarray, Pandas, and, of course, Dask.

It's a great, underutilized Python library that I've come to love after using it almost daily over the past few years.

**Comment by jacobtomlinson** [comment]
@v2thegreat the next community meeting will be on June 6th at 10am US Central. Are you up for doing a demo then?

**Comment by v2thegreat** [comment]
That sounds good to me!

**Comment by jacobtomlinson** [comment]
Ok great I've added you to the [agenda](https://docs.google.com/document/d/1UqNAP87a56ERH_xkQsS5Q_0PKYybd5Lj2WANy_hRzI0/edit#heading=h.v3srtdzg8dgm)

**Comment by scharlottej13** [comment]
> the next community meeting will be on June 6th at 10am US Central.

@ahuang11 would you also be up for doing a demo on [streamjoy](https://ahuang11.github.io/streamjoy/) at the next community meeting? (I think you ended up not presenting in April because not enough people signed up)

**Comment by ahuang11** [comment]
Yes that works for me!

**Comment by scharlottej13** [comment]
Great! I've added you to the [agenda](https://docs.google.com/document/d/1UqNAP87a56ERH_xkQsS5Q_0PKYybd5Lj2WANy_hRzI0/edit#heading=h.v3srtdzg8dgm) as well.

**Comment by raybellwaves** [comment]
Is the meeting this month going to be postponed? It falls on July 4th when people in the US have the day off work.

Note: I'll be happy to present on a dask-repo-explorer (if I finish it before the meeting)

**Comment by jacobtomlinson** [comment]
Yep we are skipping July (#383) but it would be great to have you present in August.

**Comment by jrbourbeau** [comment]
Yeah, +1 for you presenting in August if you're game @raybellwaves 

> (if I finish it before the meeting)

That's too high a bar for demo day : )

**Comment by scharlottej13** [comment]
Looking forward to seeing folks tomorrow! Just a heads up, I updated the zoom link (for tomorrow and future meetings): https://us06web.zoom.us/j/89383035703?pwd=WkRJSzNnRTh4T2R1ZjJuVVdJWlMxQT09

@raybellwaves are you still up for demoing a dask-repo-explorer?

**Comment by lucascolley** [comment]
Hi! If there is time, I would be happy to do a 2 minute demo of support for `dask.array` in SciPy.

**Comment by jacobtomlinson** [comment]
@lucascolley it would be great to have your demo in the September meeting!

**Comment by dougbrn** [comment]
Hi there! I'm with [LINCC-Frameworks](https://lsstdiscoveryalliance.org/programs/lincc-frameworks/), an organization that has been focused on enabling astronomy workflows at scale, particularly with the upcoming Rubin Observatory providing a ~petabyte-scale dataset for the community. We have been building a few projects on top of Dask, particularly [Nested-Dask](https://nested-dask.readthedocs.io/en/latest/index.html) and [LSDB](https://lsdb.readthedocs.io/en/stable/). We'd be interested in giving a small demo (~10 minutes?) at the next available meeting introducing our work with Dask. We'd love to have the opportunity to meet some of the Dask developers and share ideas on scalability challenges with the package. Thank you in advance!

**Comment by jacobtomlinson** [comment]
That sounds great @dougbrn ! The next community meeting is on the 5th of September. https://docs.dask.org/en/latest/support.html

**Comment by dougbrn** [comment]
@jacobtomlinson sounds good! I can plan to bring a ~10 minute demo to the September meeting.

**Comment by scharlottej13** [comment]
Thanks @dchudz for volunteering to demo the [LLM chatbot](https://github.com/dask/dask/pull/11556) for the next meeting on Dec. 5th! (I added it to the [agenda](https://docs.google.com/document/d/1UqNAP87a56ERH_xkQsS5Q_0PKYybd5Lj2WANy_hRzI0/edit?tab=t.0#heading=h.ht0huamr0bvb))
